hazelcast/src/main/java/com/hazelcast/nio/tcp/TcpIpConnectionManager.java

```diff
@@ -512,6 +512,7 @@ public synchronized void stop() {
         connectionsMap.clear();
         monitors.clear();
         activeConnections.clear();
+        scheduler.shutdownNow();
```

Comment:
I believe that currently there is already an issue with the threading model since it is shutdown in the stop method and it can't be restarted.